sql/src/main/java/com/rilldata/SqlConverterEntrypoint.java

```diff
@@ -42,10 +40,9 @@ public static Requests.Response transpile(Requests.Request r) {
           .setTranspileResponse(Requests.TranspileResponse.newBuilder().setSql(transpiledSql).build())
           .build();
     } catch (Exception e) {
-      e.printStackTrace();
       return Requests.Response
           .newBuilder()
-          .setError(Requests.Error.newBuilder().setMessage(e.toString()).build())
+          .setError(Requests.Error.newBuilder().setMessage(stackTraceToString(e)).build())
```

Comment:
I'm realising that this sets the stack trace as the error. We still need to also get the error message, I was picturing something like:

```java
Requests.Error.newBuilder().setMessage(e.toString()).setStackTrace(stackTraceToString(e))
```

Reply:
That stacktrace contains the error message, i decided it would be redundant to add it in a separate field, but i can change, of course

Reply:
Would be nice to add it as a separate field so the runtime doesn't have to regex the Java stacktrace :)

Reply:
Nevermind, I'll make the change, editing some code there anyway